tests/functional/contribute/test_signup.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ...pages.contribute.signup import ContributeSignUpPage
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_toggle_category_select(base_url, selenium):
+    page = ContributeSignUpPage(base_url, selenium).open()
+    category = page.contributor_categories[0]  # Select coding category
+    area = page.category_areas[0]  # Coding category areas
+    category.select_category
+    assert area.changes_to_required_field
+    assert area.is_displayed
+    category = page.contributor_categories[4]  # Select helping category
+    category.select_category
+    area.wait_to_collapse()
+    assert not area.is_required
+    assert not area.is_displayed
```

Comment:
I did wonder if this was perhaps too long also, will take this on board and see what I can do :+1: